lib/msal-browser/src/cache/CookieStorage.ts

```diff
@@ -24,6 +32,10 @@ export class CookieStorage implements IWindowStorage<string> {
         return "";
     }
 
+    getUserData(): string | null {
```

Comment:
`User` can be understood different ways. Prefer `accountData`

Reply:
A user can have more than one account, which is why we have used "account" up to this point. In this particular instance I do mean "user", not "account", because we're referring to the user, not an arbitrary single account instance. Hope that makes sense.